docs/api/alerting/create_rule.asciidoc

```diff
@@ -6,6 +6,8 @@
 
 Create {kib} rules.
 
+WARNING: This API only supports <<token-api-authentication>> and does not support <<key-authentication>>.
```

Comment:
Is it important to mention what is not supported, or can you simply say:

This API only supports Token-based authentication only.

Reply:
Updated!